argus/client/windows.py

```diff
@@ -124,7 +124,21 @@ def _run_command(protocol_client, shell_id, command,
     def _run_commands(self, commands, commands_type=util.POWERSHELL,
                       upper_timeout=CONFIG.argus.upper_timeout):
         protocol_client = self._get_protocol()
-        shell_id = protocol_client.open_shell(codepage=CODEPAGE_UTF8)
+        shell_id = None
+        open_shell_count = CONFIG.argus.retry_count
+
+        while True:
```

Comment:
I think it s time to put in place a retry method with a lambda param like in cloudbase-init